extensions/BMO/Extension.pm

```diff
@@ -2522,7 +2524,36 @@ sub _check_default_product_security_group {
 sub install_filesystem {
     my ($self, $args) = @_;
     my $files = $args->{files};
+    my $create_files = $args->{create_files};
     my $extensions_dir = bz_locations()->{extensionsdir};
+    $create_files->{__lbheartbeat__} = {
+        perms    => Bugzilla::Install::Filesystem::WS_SERVE,
+        contents => '',
```

Comment:
I strongly encourage this to contain a valid string that can be used to confirm that the load balancer got a valid response back, and not just "an empty response" (which could be any number of causes).